Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8500,19 +8500,28 @@ def ip_command(ips, reliability):
 
 
 def whois_command(reliability):
-    query = demisto.args().get('query')
-    is_recursive = argToBoolean(demisto.args().get('recursive'))
+    args = demisto.args()
+    query = args.get('query')
+    is_recursive = argToBoolean(args.get('recursive'))
+    verbose = argToBoolean(args.get('verbose'))
```

Comment:
Add the default here as well. In some cases, a customer may use `demisto.executeCommand()` in a script and the defaults won't get populated.

Reply:
Done